index.d.ts

```diff
@@ -0,0 +1,116 @@
+declare module 'cleverpush-react-native' {
+	export default class CleverPush {
```

Comment:
default or named export?
Doesn't TS prefer named exports? 
I might be wrong though